app/utility/base_world.py

```diff
@@ -36,9 +36,10 @@ def clear_config():
     @staticmethod
     def get_config(prop=None, name=None):
         name = name if name else 'main'
-        if prop:
-            return BaseWorld._app_configuration[name].get(prop)
-        return BaseWorld._app_configuration[name]
+        if name in BaseWorld._app_configuration:
```

Comment:
For context, this change means we can query for a field in _app_configuration that doesn't exist without causing everything to come crashing down. It's basically a safe check mechanism.